source/gui/settingsDialogs.py

```diff
@@ -1703,6 +1702,19 @@ def makeSettings(self, settingsSizer):
 		self.autoDialectSwitchingCheckbox.SetValue(
 			config.conf["speech"]["autoDialectSwitching"],
 		)
+		# Translators: This is the label for a checkbox in the
+		# voice settings panel (if checked, the language switched to will be reported).
+		reportLanguageText = _("Report language when automatic switching is enabled")
```

Comment:
Would you add an accelerator (ampersand) as for other items?

Reply:
I think we should add an accelerator here. The automatic language switching checkbox doesn't have one, and I think that this new option won't be almost used. At least for me it's too verbose and I prefer to consult the language assigning a command to the new script.